lib/modules/apostrophe-logger/index.js

```diff
@@ -1,18 +1,21 @@
-const config = require('./config/index');
+const LogProvider = require('./lib/logProvider').LogProvider;
 
 module.exports = {
   alias: 'logger',
+  logProvider: null,
   construct: (self, options) => {
-    const clientLoggingProvider = process.env.LOGGING_PROVIDER_CLIENT;
-    const providerConfig = config[clientLoggingProvider];
+    const clientLoggingProvider = process.env.LOG_PROVIDER_CLIENT;
```

Comment:
Ik denk dat het logischer is om deze `const logProviderClient` te noemen, dan komt het overeen met de naamgeving vanuit de env en van de `logProvider` functie